public_app/components/Nav.js

```diff
@@ -0,0 +1,19 @@
+import React    from 'react';
+var NavLink = require('react-router-dom').NavLink;
+
+function Nav (){
+    return (
+        <div className='NavContainer'>
```

Comment:
Css class names are case insensitive so never use capitalization in css class names, use dash `-` or underscore `_`